### ISSUE TITLE
pkg/golinters/dupl incorrect for long arrays of hex numbers (rare edge case)

### ISSUE THREAD

**Issue by y3sh** [golangci/golangci-lint#960]
I've been translating some older crypto hashing algorithms, which I think has surfaced an edge case for the dup lint check. It's common to initialize a hashing algorithms lookup tables. In the case of the old [whirlpool hash](https://github.com/gpg/libgcrypt/blob/master/cipher/whirlpool.c), initializing each lookup table yields errors in the dup checker in go. See go sample at the bottom.

- [x] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [x] Yes, I've searched similar issues on GitHub and didn't find any.
- [x] Yes, I've included all information below (version, config, etc).

Please include the following information:

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
golangci-lint has version 1.23.3 built from 2241adf on 2020-02-04T04:58:40Z
```

</details>

<details><summary>Config file</summary>

```console
$ cat .golangci.yml
linters-settings:
  govet:
    check-shadowing: true
    settings:
      printf:
        funcs:
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
  golint:
    min-confidence: 0
  gocyclo:
    min-complexity: 10
  maligned:
    suggest-new: true
  dupl:
    threshold: 120
  goconst:
    min-len: 2
    min-occurrences: 2
  depguard:
    list-type: blacklist
    packages:
    # - foo/bar
  misspell:
    locale: US
  lll:
    line-length: 140
  goimports:
    local-prefixes: github.com/golangci/golangci-lint
  gocritic:
    enabled-tags:
      - performance
      - style
      - experimental
    disabled-checks:
      - wrapperFunc
      - dupImport # https://github.com/go-critic/go-critic/issues/845

linters:
  enable-all: true
  disable:
    - maligned
    - prealloc
    - gochecknoglobals

run:
  tests: false

  skip-dirs:
    - test/testdata_etc
    - pkg/golinters/goanalysis/(checker|passes)

issues:
  exclude-rules:
    - text: "weak cryptographic primitive"
      linters:
        - gosec

# golangci.com configuration
# https://github.com/golangci/golangci/wiki/Configuration
service:
  golangci-lint-version: 1.17.x # use the fixed version to not introduce new linters unexpectedly
  prepare:
    - echo "Prepping golangci"

```

</details>

<details><summary>Go environment</summary>

```console
$ go version && go env
GO111MODULE="on"
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/.../code/.gocache"
GOENV="/Users/.../Library/Application Support/go/env"
GOEXE=""
GOFLAGS=" -mod="
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/.../code/go"
GOPROXY="direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/ym/123k1xzn2q3005dh0qrryrx40000gn/T/go-build494128655=/tmp/go-build -gno-record-gcc-switches -fno-common"

```

</details>

<details><summary>Verbose output of running</summary>

```console
$ golangci-lint run -v

dup.go:3: 3-132 lines are duplicate of `dup.go:134-263` (dupl)
var a = [256]uint64{
        0x18186018c07830d8, 0x23238c2305af4626,
        0xc6c63fc67ef991b8, 0xe8e887e8136fcdfb,
        0x878726874ca113cb, 0xb8b8dab8a9626d11,
        0x0101040108050209, 0x4f4f214f426e9e0d,
        0x3636d836adee6c9b, 0xa6a6a2a6590451ff,
        0xd2d26fd2debdb90c, 0xf5f5f3f5fb06f70e,
        0x7979f979ef80f296, 0x6f6fa16f5fcede30,
        0x91917e91fcef3f6d, 0x52525552aa07a4f8,
        0x60609d6027fdc047, 0xbcbccabc89766535,
        0x9b9b569baccd2b37, 0x8e8e028e048c018a,
        0xa3a3b6a371155bd2, 0x0c0c300c603c186c,
        0x7b7bf17bff8af684, 0x3535d435b5e16a80,
        0x1d1d741de8693af5, 0xe0e0a7e05347ddb3,
        0xd7d77bd7f6acb321, 0xc2c22fc25eed999c,
        0x2e2eb82e6d965c43, 0x4b4b314b627a9629,
        0xfefedffea321e15d, 0x575741578216aed5,
        0x15155415a8412abd, 0x7777c1779fb6eee8,
        0x3737dc37a5eb6e92, 0xe5e5b3e57b56d79e,
        0x9f9f469f8cd92313, 0xf0f0e7f0d317fd23,
        0x4a4a354a6a7f9420, 0xdada4fda9e95a944,
        0x58587d58fa25b0a2, 0xc9c903c906ca8fcf,
        0x2929a429558d527c, 0x0a0a280a5022145a,
        0xb1b1feb1e14f7f50, 0xa0a0baa0691a5dc9,
        0x6b6bb16b7fdad614, 0x85852e855cab17d9,
        0xbdbdcebd8173673c, 0x5d5d695dd234ba8f,
        0x1010401080502090, 0xf4f4f7f4f303f507,
        0xcbcb0bcb16c08bdd, 0x3e3ef83eedc67cd3,
        0x0505140528110a2d, 0x676781671fe6ce78,
        0xe4e4b7e47353d597, 0x27279c2725bb4e02,
        0x4141194132588273, 0x8b8b168b2c9d0ba7,
        0xa7a7a6a7510153f6, 0x7d7de97dcf94fab2,
        0x95956e95dcfb3749, 0xd8d847d88e9fad56,
        0xfbfbcbfb8b30eb70, 0xeeee9fee2371c1cd,
        0x7c7ced7cc791f8bb, 0x6666856617e3cc71,
        0xdddd53dda68ea77b, 0x17175c17b84b2eaf,
        0x4747014702468e45, 0x9e9e429e84dc211a,
        0xcaca0fca1ec589d4, 0x2d2db42d75995a58,
        0xbfbfc6bf9179632e, 0x07071c07381b0e3f,
        0xadad8ead012347ac, 0x5a5a755aea2fb4b0,
        0x838336836cb51bef, 0x3333cc3385ff66b6,
        0x636391633ff2c65c, 0x02020802100a0412,
        0xaaaa92aa39384993, 0x7171d971afa8e2de,
        0xc8c807c80ecf8dc6, 0x19196419c87d32d1,
        0x494939497270923b, 0xd9d943d9869aaf5f,
        0xf2f2eff2c31df931, 0xe3e3abe34b48dba8,
        0x5b5b715be22ab6b9, 0x88881a8834920dbc,
        0x9a9a529aa4c8293e, 0x262698262dbe4c0b,
        0x3232c8328dfa64bf, 0xb0b0fab0e94a7d59,
        0xe9e983e91b6acff2, 0x0f0f3c0f78331e77,
        0xd5d573d5e6a6b733, 0x80803a8074ba1df4,
        0xbebec2be997c6127, 0xcdcd13cd26de87eb,
        0x3434d034bde46889, 0x48483d487a759032,
        0xffffdbffab24e354, 0x7a7af57af78ff48d,
        0x90907a90f4ea3d64, 0x5f5f615fc23ebe9d,
        0x202080201da0403d, 0x6868bd6867d5d00f,
        0x1a1a681ad07234ca, 0xaeae82ae192c41b7,
        0xb4b4eab4c95e757d, 0x54544d549a19a8ce,
        0x93937693ece53b7f, 0x222288220daa442f,
        0x64648d6407e9c863, 0xf1f1e3f1db12ff2a,
        0x7373d173bfa2e6cc, 0x12124812905a2482,
        0x40401d403a5d807a, 0x0808200840281048,
        0xc3c32bc356e89b95, 0xecec97ec337bc5df,
        0xdbdb4bdb9690ab4d, 0xa1a1bea1611f5fc0,
        0x8d8d0e8d1c830791, 0x3d3df43df5c97ac8,
        0x97976697ccf1335b, 0x0000000000000000,
        0xcfcf1bcf36d483f9, 0x2b2bac2b4587566e,
        0x7676c57697b3ece1, 0x8282328264b019e6,
        0xd6d67fd6fea9b128, 0x1b1b6c1bd87736c3,
        0xb5b5eeb5c15b7774, 0xafaf86af112943be,
        0x6a6ab56a77dfd41d, 0x50505d50ba0da0ea,
        0x45450945124c8a57, 0xf3f3ebf3cb18fb38,
        0x3030c0309df060ad, 0xefef9bef2b74c3c4,
        0x3f3ffc3fe5c37eda, 0x55554955921caac7,
        0xa2a2b2a2791059db, 0xeaea8fea0365c9e9,
        0x656589650fecca6a, 0xbabad2bab9686903,
        0x2f2fbc2f65935e4a, 0xc0c027c04ee79d8e,
        0xdede5fdebe81a160, 0x1c1c701ce06c38fc,
        0xfdfdd3fdbb2ee746, 0x4d4d294d52649a1f,
        0x92927292e4e03976, 0x7575c9758fbceafa,
        0x06061806301e0c36, 0x8a8a128a249809ae,
        0xb2b2f2b2f940794b, 0xe6e6bfe66359d185,
        0x0e0e380e70361c7e, 0x1f1f7c1ff8633ee7,
        0x6262956237f7c455, 0xd4d477d4eea3b53a,
        0xa8a89aa829324d81, 0x96966296c4f43152,
        0xf9f9c3f99b3aef62, 0xc5c533c566f697a3,
        0x2525942535b14a10, 0x59597959f220b2ab,
        0x84842a8454ae15d0, 0x7272d572b7a7e4c5,
        0x3939e439d5dd72ec, 0x4c4c2d4c5a619816,
        0x5e5e655eca3bbc94, 0x7878fd78e785f09f,
        0x3838e038ddd870e5, 0x8c8c0a8c14860598,
        0xd1d163d1c6b2bf17, 0xa5a5aea5410b57e4,
        0xe2e2afe2434dd9a1, 0x616199612ff8c24e,
        0xb3b3f6b3f1457b42, 0x2121842115a54234,
        0x9c9c4a9c94d62508, 0x1e1e781ef0663cee,
        0x4343114322528661, 0xc7c73bc776fc93b1,
        0xfcfcd7fcb32be54f, 0x0404100420140824,
        0x51515951b208a2e3, 0x99995e99bcc72f25,
        0x6d6da96d4fc4da22, 0x0d0d340d68391a65,
        0xfafacffa8335e979, 0xdfdf5bdfb684a369,
        0x7e7ee57ed79bfca9, 0x242490243db44819,
        0x3b3bec3bc5d776fe, 0xabab96ab313d4b9a,
        0xcece1fce3ed181f0, 0x1111441188552299,
        0x8f8f068f0c890383, 0x4e4e254e4a6b9c04,
        0xb7b7e6b7d1517366, 0xebeb8beb0b60cbe0,
        0x3c3cf03cfdcc78c1, 0x81813e817cbf1ffd,
        0x94946a94d4fe3540, 0xf7f7fbf7eb0cf31c,
        0xb9b9deb9a1676f18, 0x13134c13985f268b,
        0x2c2cb02c7d9c5851, 0xd3d36bd3d6b8bb05,
        0xe7e7bbe76b5cd38c, 0x6e6ea56e57cbdc39,
        0xc4c437c46ef395aa, 0x03030c03180f061b,
        0x565645568a13acdc, 0x44440d441a49885e,
        0x7f7fe17fdf9efea0, 0xa9a99ea921374f88,
        0x2a2aa82a4d825467, 0xbbbbd6bbb16d6b0a,
        0xc1c123c146e29f87, 0x53535153a202a6f1,
        0xdcdc57dcae8ba572, 0x0b0b2c0b58271653,
        0x9d9d4e9d9cd32701, 0x6c6cad6c47c1d82b,
        0x3131c43195f562a4, 0x7474cd7487b9e8f3,
        0xf6f6fff6e309f115, 0x464605460a438c4c,
        0xacac8aac092645a5, 0x89891e893c970fb5,
        0x14145014a04428b4, 0xe1e1a3e15b42dfba,
        0x16165816b04e2ca6, 0x3a3ae83acdd274f7,
        0x6969b9696fd0d206, 0x09092409482d1241,
        0x7070dd70a7ade0d7, 0xb6b6e2b6d954716f,
        0xd0d067d0ceb7bd1e, 0xeded93ed3b7ec7d6,
        0xcccc17cc2edb85e2, 0x424215422a578468,
        0x98985a98b4c22d2c, 0xa4a4aaa4490e55ed,
        0x2828a0285d885075, 0x5c5c6d5cda31b886,
        0xf8f8c7f8933fed6b, 0x8686228644a411c2,
}
dup.go:134: 134-263 lines are duplicate of `dup.go:3-132` (dupl)
var b = [256]uint64{
        0x186018c07830d818, 0x238c2305af462623,
        0xc63fc67ef991b8c6, 0xe887e8136fcdfbe8,
        0x8726874ca113cb87, 0xb8dab8a9626d11b8,
        0x0104010805020901, 0x4f214f426e9e0d4f,
        0x36d836adee6c9b36, 0xa6a2a6590451ffa6,
        0xd26fd2debdb90cd2, 0xf5f3f5fb06f70ef5,
        0x79f979ef80f29679, 0x6fa16f5fcede306f,
        0x917e91fcef3f6d91, 0x525552aa07a4f852,
        0x609d6027fdc04760, 0xbccabc89766535bc,
        0x9b569baccd2b379b, 0x8e028e048c018a8e,
        0xa3b6a371155bd2a3, 0x0c300c603c186c0c,
        0x7bf17bff8af6847b, 0x35d435b5e16a8035,
        0x1d741de8693af51d, 0xe0a7e05347ddb3e0,
        0xd77bd7f6acb321d7, 0xc22fc25eed999cc2,
        0x2eb82e6d965c432e, 0x4b314b627a96294b,
        0xfedffea321e15dfe, 0x5741578216aed557,
        0x155415a8412abd15, 0x77c1779fb6eee877,
        0x37dc37a5eb6e9237, 0xe5b3e57b56d79ee5,
        0x9f469f8cd923139f, 0xf0e7f0d317fd23f0,
        0x4a354a6a7f94204a, 0xda4fda9e95a944da,
        0x587d58fa25b0a258, 0xc903c906ca8fcfc9,
        0x29a429558d527c29, 0x0a280a5022145a0a,
        0xb1feb1e14f7f50b1, 0xa0baa0691a5dc9a0,
        0x6bb16b7fdad6146b, 0x852e855cab17d985,
        0xbdcebd8173673cbd, 0x5d695dd234ba8f5d,
        0x1040108050209010, 0xf4f7f4f303f507f4,
        0xcb0bcb16c08bddcb, 0x3ef83eedc67cd33e,
        0x05140528110a2d05, 0x6781671fe6ce7867,
        0xe4b7e47353d597e4, 0x279c2725bb4e0227,
        0x4119413258827341, 0x8b168b2c9d0ba78b,
        0xa7a6a7510153f6a7, 0x7de97dcf94fab27d,
        0x956e95dcfb374995, 0xd847d88e9fad56d8,
        0xfbcbfb8b30eb70fb, 0xee9fee2371c1cdee,
        0x7ced7cc791f8bb7c, 0x66856617e3cc7166,
        0xdd53dda68ea77bdd, 0x175c17b84b2eaf17,
        0x47014702468e4547, 0x9e429e84dc211a9e,
        0xca0fca1ec589d4ca, 0x2db42d75995a582d,
        0xbfc6bf9179632ebf, 0x071c07381b0e3f07,
        0xad8ead012347acad, 0x5a755aea2fb4b05a,
        0x8336836cb51bef83, 0x33cc3385ff66b633,
        0x6391633ff2c65c63, 0x020802100a041202,
        0xaa92aa39384993aa, 0x71d971afa8e2de71,
        0xc807c80ecf8dc6c8, 0x196419c87d32d119,
        0x4939497270923b49, 0xd943d9869aaf5fd9,
        0xf2eff2c31df931f2, 0xe3abe34b48dba8e3,
        0x5b715be22ab6b95b, 0x881a8834920dbc88,
        0x9a529aa4c8293e9a, 0x2698262dbe4c0b26,
        0x32c8328dfa64bf32, 0xb0fab0e94a7d59b0,
        0xe983e91b6acff2e9, 0x0f3c0f78331e770f,
        0xd573d5e6a6b733d5, 0x803a8074ba1df480,
        0xbec2be997c6127be, 0xcd13cd26de87ebcd,
        0x34d034bde4688934, 0x483d487a75903248,
        0xffdbffab24e354ff, 0x7af57af78ff48d7a,
        0x907a90f4ea3d6490, 0x5f615fc23ebe9d5f,
        0x2080201da0403d20, 0x68bd6867d5d00f68,
        0x1a681ad07234ca1a, 0xae82ae192c41b7ae,
        0xb4eab4c95e757db4, 0x544d549a19a8ce54,
        0x937693ece53b7f93, 0x2288220daa442f22,
        0x648d6407e9c86364, 0xf1e3f1db12ff2af1,
        0x73d173bfa2e6cc73, 0x124812905a248212,
        0x401d403a5d807a40, 0x0820084028104808,
        0xc32bc356e89b95c3, 0xec97ec337bc5dfec,
        0xdb4bdb9690ab4ddb, 0xa1bea1611f5fc0a1,
        0x8d0e8d1c8307918d, 0x3df43df5c97ac83d,
        0x976697ccf1335b97, 0x0000000000000000,
        0xcf1bcf36d483f9cf, 0x2bac2b4587566e2b,
        0x76c57697b3ece176, 0x82328264b019e682,
        0xd67fd6fea9b128d6, 0x1b6c1bd87736c31b,
        0xb5eeb5c15b7774b5, 0xaf86af112943beaf,
        0x6ab56a77dfd41d6a, 0x505d50ba0da0ea50,
        0x450945124c8a5745, 0xf3ebf3cb18fb38f3,
        0x30c0309df060ad30, 0xef9bef2b74c3c4ef,
        0x3ffc3fe5c37eda3f, 0x554955921caac755,
        0xa2b2a2791059dba2, 0xea8fea0365c9e9ea,
        0x6589650fecca6a65, 0xbad2bab9686903ba,
        0x2fbc2f65935e4a2f, 0xc027c04ee79d8ec0,
        0xde5fdebe81a160de, 0x1c701ce06c38fc1c,
        0xfdd3fdbb2ee746fd, 0x4d294d52649a1f4d,
        0x927292e4e0397692, 0x75c9758fbceafa75,
        0x061806301e0c3606, 0x8a128a249809ae8a,
        0xb2f2b2f940794bb2, 0xe6bfe66359d185e6,
        0x0e380e70361c7e0e, 0x1f7c1ff8633ee71f,
        0x62956237f7c45562, 0xd477d4eea3b53ad4,
        0xa89aa829324d81a8, 0x966296c4f4315296,
        0xf9c3f99b3aef62f9, 0xc533c566f697a3c5,
        0x25942535b14a1025, 0x597959f220b2ab59,
        0x842a8454ae15d084, 0x72d572b7a7e4c572,
        0x39e439d5dd72ec39, 0x4c2d4c5a6198164c,
        0x5e655eca3bbc945e, 0x78fd78e785f09f78,
        0x38e038ddd870e538, 0x8c0a8c148605988c,
        0xd163d1c6b2bf17d1, 0xa5aea5410b57e4a5,
        0xe2afe2434dd9a1e2, 0x6199612ff8c24e61,
        0xb3f6b3f1457b42b3, 0x21842115a5423421,
        0x9c4a9c94d625089c, 0x1e781ef0663cee1e,
        0x4311432252866143, 0xc73bc776fc93b1c7,
        0xfcd7fcb32be54ffc, 0x0410042014082404,
        0x515951b208a2e351, 0x995e99bcc72f2599,
        0x6da96d4fc4da226d, 0x0d340d68391a650d,
        0xfacffa8335e979fa, 0xdf5bdfb684a369df,
        0x7ee57ed79bfca97e, 0x2490243db4481924,
        0x3bec3bc5d776fe3b, 0xab96ab313d4b9aab,
        0xce1fce3ed181f0ce, 0x1144118855229911,
        0x8f068f0c8903838f, 0x4e254e4a6b9c044e,
        0xb7e6b7d1517366b7, 0xeb8beb0b60cbe0eb,
        0x3cf03cfdcc78c13c, 0x813e817cbf1ffd81,
        0x946a94d4fe354094, 0xf7fbf7eb0cf31cf7,
        0xb9deb9a1676f18b9, 0x134c13985f268b13,
        0x2cb02c7d9c58512c, 0xd36bd3d6b8bb05d3,
        0xe7bbe76b5cd38ce7, 0x6ea56e57cbdc396e,
        0xc437c46ef395aac4, 0x030c03180f061b03,
        0x5645568a13acdc56, 0x440d441a49885e44,
        0x7fe17fdf9efea07f, 0xa99ea921374f88a9,
        0x2aa82a4d8254672a, 0xbbd6bbb16d6b0abb,
        0xc123c146e29f87c1, 0x535153a202a6f153,
        0xdc57dcae8ba572dc, 0x0b2c0b582716530b,
        0x9d4e9d9cd327019d, 0x6cad6c47c1d82b6c,
        0x31c43195f562a431, 0x74cd7487b9e8f374,
        0xf6fff6e309f115f6, 0x4605460a438c4c46,
        0xac8aac092645a5ac, 0x891e893c970fb589,
        0x145014a04428b414, 0xe1a3e15b42dfbae1,
        0x165816b04e2ca616, 0x3ae83acdd274f73a,
        0x69b9696fd0d20669, 0x092409482d124109,
        0x70dd70a7ade0d770, 0xb6e2b6d954716fb6,
        0xd067d0ceb7bd1ed0, 0xed93ed3b7ec7d6ed,
        0xcc17cc2edb85e2cc, 0x4215422a57846842,
        0x985a98b4c22d2c98, 0xa4aaa4490e55eda4,
        0x28a0285d88507528, 0x5c6d5cda31b8865c,
        0xf8c7f8933fed6bf8, 0x86228644a411c286,
}


```

</details>

Reproducing go sample:

```go
package main

var a = [256]uint64{
	0x18186018c07830d8, 0x23238c2305af4626,
	0xc6c63fc67ef991b8, 0xe8e887e8136fcdfb,
	0x878726874ca113cb, 0xb8b8dab8a9626d11,
	0x0101040108050209, 0x4f4f214f426e9e0d,
	0x3636d836adee6c9b, 0xa6a6a2a6590451ff,
	0xd2d26fd2debdb90c, 0xf5f5f3f5fb06f70e,
	0x7979f979ef80f296, 0x6f6fa16f5fcede30,
	0x91917e91fcef3f6d, 0x52525552aa07a4f8,
	0x60609d6027fdc047, 0xbcbccabc89766535,
	0x9b9b569baccd2b37, 0x8e8e028e048c018a,
	0xa3a3b6a371155bd2, 0x0c0c300c603c186c,
	0x7b7bf17bff8af684, 0x3535d435b5e16a80,
	0x1d1d741de8693af5, 0xe0e0a7e05347ddb3,
	0xd7d77bd7f6acb321, 0xc2c22fc25eed999c,
	0x2e2eb82e6d965c43, 0x4b4b314b627a9629,
	0xfefedffea321e15d, 0x575741578216aed5,
	0x15155415a8412abd, 0x7777c1779fb6eee8,
	0x3737dc37a5eb6e92, 0xe5e5b3e57b56d79e,
	0x9f9f469f8cd92313, 0xf0f0e7f0d317fd23,
	0x4a4a354a6a7f9420, 0xdada4fda9e95a944,
	0x58587d58fa25b0a2, 0xc9c903c906ca8fcf,
	0x2929a429558d527c, 0x0a0a280a5022145a,
	0xb1b1feb1e14f7f50, 0xa0a0baa0691a5dc9,
	0x6b6bb16b7fdad614, 0x85852e855cab17d9,
	0xbdbdcebd8173673c, 0x5d5d695dd234ba8f,
	0x1010401080502090, 0xf4f4f7f4f303f507,
	0xcbcb0bcb16c08bdd, 0x3e3ef83eedc67cd3,
	0x0505140528110a2d, 0x676781671fe6ce78,
	0xe4e4b7e47353d597, 0x27279c2725bb4e02,
	0x4141194132588273, 0x8b8b168b2c9d0ba7,
	0xa7a7a6a7510153f6, 0x7d7de97dcf94fab2,
	0x95956e95dcfb3749, 0xd8d847d88e9fad56,
	0xfbfbcbfb8b30eb70, 0xeeee9fee2371c1cd,
	0x7c7ced7cc791f8bb, 0x6666856617e3cc71,
	0xdddd53dda68ea77b, 0x17175c17b84b2eaf,
	0x4747014702468e45, 0x9e9e429e84dc211a,
	0xcaca0fca1ec589d4, 0x2d2db42d75995a58,
	0xbfbfc6bf9179632e, 0x07071c07381b0e3f,
	0xadad8ead012347ac, 0x5a5a755aea2fb4b0,
	0x838336836cb51bef, 0x3333cc3385ff66b6,
	0x636391633ff2c65c, 0x02020802100a0412,
	0xaaaa92aa39384993, 0x7171d971afa8e2de,
	0xc8c807c80ecf8dc6, 0x19196419c87d32d1,
	0x494939497270923b, 0xd9d943d9869aaf5f,
	0xf2f2eff2c31df931, 0xe3e3abe34b48dba8,
	0x5b5b715be22ab6b9, 0x88881a8834920dbc,
	0x9a9a529aa4c8293e, 0x262698262dbe4c0b,
	0x3232c8328dfa64bf, 0xb0b0fab0e94a7d59,
	0xe9e983e91b6acff2, 0x0f0f3c0f78331e77,
	0xd5d573d5e6a6b733, 0x80803a8074ba1df4,
	0xbebec2be997c6127, 0xcdcd13cd26de87eb,
	0x3434d034bde46889, 0x48483d487a759032,
	0xffffdbffab24e354, 0x7a7af57af78ff48d,
	0x90907a90f4ea3d64, 0x5f5f615fc23ebe9d,
	0x202080201da0403d, 0x6868bd6867d5d00f,
	0x1a1a681ad07234ca, 0xaeae82ae192c41b7,
	0xb4b4eab4c95e757d, 0x54544d549a19a8ce,
	0x93937693ece53b7f, 0x222288220daa442f,
	0x64648d6407e9c863, 0xf1f1e3f1db12ff2a,
	0x7373d173bfa2e6cc, 0x12124812905a2482,
	0x40401d403a5d807a, 0x0808200840281048,
	0xc3c32bc356e89b95, 0xecec97ec337bc5df,
	0xdbdb4bdb9690ab4d, 0xa1a1bea1611f5fc0,
	0x8d8d0e8d1c830791, 0x3d3df43df5c97ac8,
	0x97976697ccf1335b, 0x0000000000000000,
	0xcfcf1bcf36d483f9, 0x2b2bac2b4587566e,
	0x7676c57697b3ece1, 0x8282328264b019e6,
	0xd6d67fd6fea9b128, 0x1b1b6c1bd87736c3,
	0xb5b5eeb5c15b7774, 0xafaf86af112943be,
	0x6a6ab56a77dfd41d, 0x50505d50ba0da0ea,
	0x45450945124c8a57, 0xf3f3ebf3cb18fb38,
	0x3030c0309df060ad, 0xefef9bef2b74c3c4,
	0x3f3ffc3fe5c37eda, 0x55554955921caac7,
	0xa2a2b2a2791059db, 0xeaea8fea0365c9e9,
	0x656589650fecca6a, 0xbabad2bab9686903,
	0x2f2fbc2f65935e4a, 0xc0c027c04ee79d8e,
	0xdede5fdebe81a160, 0x1c1c701ce06c38fc,
	0xfdfdd3fdbb2ee746, 0x4d4d294d52649a1f,
	0x92927292e4e03976, 0x7575c9758fbceafa,
	0x06061806301e0c36, 0x8a8a128a249809ae,
	0xb2b2f2b2f940794b, 0xe6e6bfe66359d185,
	0x0e0e380e70361c7e, 0x1f1f7c1ff8633ee7,
	0x6262956237f7c455, 0xd4d477d4eea3b53a,
	0xa8a89aa829324d81, 0x96966296c4f43152,
	0xf9f9c3f99b3aef62, 0xc5c533c566f697a3,
	0x2525942535b14a10, 0x59597959f220b2ab,
	0x84842a8454ae15d0, 0x7272d572b7a7e4c5,
	0x3939e439d5dd72ec, 0x4c4c2d4c5a619816,
	0x5e5e655eca3bbc94, 0x7878fd78e785f09f,
	0x3838e038ddd870e5, 0x8c8c0a8c14860598,
	0xd1d163d1c6b2bf17, 0xa5a5aea5410b57e4,
	0xe2e2afe2434dd9a1, 0x616199612ff8c24e,
	0xb3b3f6b3f1457b42, 0x2121842115a54234,
	0x9c9c4a9c94d62508, 0x1e1e781ef0663cee,
	0x4343114322528661, 0xc7c73bc776fc93b1,
	0xfcfcd7fcb32be54f, 0x0404100420140824,
	0x51515951b208a2e3, 0x99995e99bcc72f25,
	0x6d6da96d4fc4da22, 0x0d0d340d68391a65,
	0xfafacffa8335e979, 0xdfdf5bdfb684a369,
	0x7e7ee57ed79bfca9, 0x242490243db44819,
	0x3b3bec3bc5d776fe, 0xabab96ab313d4b9a,
	0xcece1fce3ed181f0, 0x1111441188552299,
	0x8f8f068f0c890383, 0x4e4e254e4a6b9c04,
	0xb7b7e6b7d1517366, 0xebeb8beb0b60cbe0,
	0x3c3cf03cfdcc78c1, 0x81813e817cbf1ffd,
	0x94946a94d4fe3540, 0xf7f7fbf7eb0cf31c,
	0xb9b9deb9a1676f18, 0x13134c13985f268b,
	0x2c2cb02c7d9c5851, 0xd3d36bd3d6b8bb05,
	0xe7e7bbe76b5cd38c, 0x6e6ea56e57cbdc39,
	0xc4c437c46ef395aa, 0x03030c03180f061b,
	0x565645568a13acdc, 0x44440d441a49885e,
	0x7f7fe17fdf9efea0, 0xa9a99ea921374f88,
	0x2a2aa82a4d825467, 0xbbbbd6bbb16d6b0a,
	0xc1c123c146e29f87, 0x53535153a202a6f1,
	0xdcdc57dcae8ba572, 0x0b0b2c0b58271653,
	0x9d9d4e9d9cd32701, 0x6c6cad6c47c1d82b,
	0x3131c43195f562a4, 0x7474cd7487b9e8f3,
	0xf6f6fff6e309f115, 0x464605460a438c4c,
	0xacac8aac092645a5, 0x89891e893c970fb5,
	0x14145014a04428b4, 0xe1e1a3e15b42dfba,
	0x16165816b04e2ca6, 0x3a3ae83acdd274f7,
	0x6969b9696fd0d206, 0x09092409482d1241,
	0x7070dd70a7ade0d7, 0xb6b6e2b6d954716f,
	0xd0d067d0ceb7bd1e, 0xeded93ed3b7ec7d6,
	0xcccc17cc2edb85e2, 0x424215422a578468,
	0x98985a98b4c22d2c, 0xa4a4aaa4490e55ed,
	0x2828a0285d885075, 0x5c5c6d5cda31b886,
	0xf8f8c7f8933fed6b, 0x8686228644a411c2,
}

var b = [256]uint64{
	0x186018c07830d818, 0x238c2305af462623,
	0xc63fc67ef991b8c6, 0xe887e8136fcdfbe8,
	0x8726874ca113cb87, 0xb8dab8a9626d11b8,
	0x0104010805020901, 0x4f214f426e9e0d4f,
	0x36d836adee6c9b36, 0xa6a2a6590451ffa6,
	0xd26fd2debdb90cd2, 0xf5f3f5fb06f70ef5,
	0x79f979ef80f29679, 0x6fa16f5fcede306f,
	0x917e91fcef3f6d91, 0x525552aa07a4f852,
	0x609d6027fdc04760, 0xbccabc89766535bc,
	0x9b569baccd2b379b, 0x8e028e048c018a8e,
	0xa3b6a371155bd2a3, 0x0c300c603c186c0c,
	0x7bf17bff8af6847b, 0x35d435b5e16a8035,
	0x1d741de8693af51d, 0xe0a7e05347ddb3e0,
	0xd77bd7f6acb321d7, 0xc22fc25eed999cc2,
	0x2eb82e6d965c432e, 0x4b314b627a96294b,
	0xfedffea321e15dfe, 0x5741578216aed557,
	0x155415a8412abd15, 0x77c1779fb6eee877,
	0x37dc37a5eb6e9237, 0xe5b3e57b56d79ee5,
	0x9f469f8cd923139f, 0xf0e7f0d317fd23f0,
	0x4a354a6a7f94204a, 0xda4fda9e95a944da,
	0x587d58fa25b0a258, 0xc903c906ca8fcfc9,
	0x29a429558d527c29, 0x0a280a5022145a0a,
	0xb1feb1e14f7f50b1, 0xa0baa0691a5dc9a0,
	0x6bb16b7fdad6146b, 0x852e855cab17d985,
	0xbdcebd8173673cbd, 0x5d695dd234ba8f5d,
	0x1040108050209010, 0xf4f7f4f303f507f4,
	0xcb0bcb16c08bddcb, 0x3ef83eedc67cd33e,
	0x05140528110a2d05, 0x6781671fe6ce7867,
	0xe4b7e47353d597e4, 0x279c2725bb4e0227,
	0x4119413258827341, 0x8b168b2c9d0ba78b,
	0xa7a6a7510153f6a7, 0x7de97dcf94fab27d,
	0x956e95dcfb374995, 0xd847d88e9fad56d8,
	0xfbcbfb8b30eb70fb, 0xee9fee2371c1cdee,
	0x7ced7cc791f8bb7c, 0x66856617e3cc7166,
	0xdd53dda68ea77bdd, 0x175c17b84b2eaf17,
	0x47014702468e4547, 0x9e429e84dc211a9e,
	0xca0fca1ec589d4ca, 0x2db42d75995a582d,
	0xbfc6bf9179632ebf, 0x071c07381b0e3f07,
	0xad8ead012347acad, 0x5a755aea2fb4b05a,
	0x8336836cb51bef83, 0x33cc3385ff66b633,
	0x6391633ff2c65c63, 0x020802100a041202,
	0xaa92aa39384993aa, 0x71d971afa8e2de71,
	0xc807c80ecf8dc6c8, 0x196419c87d32d119,
	0x4939497270923b49, 0xd943d9869aaf5fd9,
	0xf2eff2c31df931f2, 0xe3abe34b48dba8e3,
	0x5b715be22ab6b95b, 0x881a8834920dbc88,
	0x9a529aa4c8293e9a, 0x2698262dbe4c0b26,
	0x32c8328dfa64bf32, 0xb0fab0e94a7d59b0,
	0xe983e91b6acff2e9, 0x0f3c0f78331e770f,
	0xd573d5e6a6b733d5, 0x803a8074ba1df480,
	0xbec2be997c6127be, 0xcd13cd26de87ebcd,
	0x34d034bde4688934, 0x483d487a75903248,
	0xffdbffab24e354ff, 0x7af57af78ff48d7a,
	0x907a90f4ea3d6490, 0x5f615fc23ebe9d5f,
	0x2080201da0403d20, 0x68bd6867d5d00f68,
	0x1a681ad07234ca1a, 0xae82ae192c41b7ae,
	0xb4eab4c95e757db4, 0x544d549a19a8ce54,
	0x937693ece53b7f93, 0x2288220daa442f22,
	0x648d6407e9c86364, 0xf1e3f1db12ff2af1,
	0x73d173bfa2e6cc73, 0x124812905a248212,
	0x401d403a5d807a40, 0x0820084028104808,
	0xc32bc356e89b95c3, 0xec97ec337bc5dfec,
	0xdb4bdb9690ab4ddb, 0xa1bea1611f5fc0a1,
	0x8d0e8d1c8307918d, 0x3df43df5c97ac83d,
	0x976697ccf1335b97, 0x0000000000000000,
	0xcf1bcf36d483f9cf, 0x2bac2b4587566e2b,
	0x76c57697b3ece176, 0x82328264b019e682,
	0xd67fd6fea9b128d6, 0x1b6c1bd87736c31b,
	0xb5eeb5c15b7774b5, 0xaf86af112943beaf,
	0x6ab56a77dfd41d6a, 0x505d50ba0da0ea50,
	0x450945124c8a5745, 0xf3ebf3cb18fb38f3,
	0x30c0309df060ad30, 0xef9bef2b74c3c4ef,
	0x3ffc3fe5c37eda3f, 0x554955921caac755,
	0xa2b2a2791059dba2, 0xea8fea0365c9e9ea,
	0x6589650fecca6a65, 0xbad2bab9686903ba,
	0x2fbc2f65935e4a2f, 0xc027c04ee79d8ec0,
	0xde5fdebe81a160de, 0x1c701ce06c38fc1c,
	0xfdd3fdbb2ee746fd, 0x4d294d52649a1f4d,
	0x927292e4e0397692, 0x75c9758fbceafa75,
	0x061806301e0c3606, 0x8a128a249809ae8a,
	0xb2f2b2f940794bb2, 0xe6bfe66359d185e6,
	0x0e380e70361c7e0e, 0x1f7c1ff8633ee71f,
	0x62956237f7c45562, 0xd477d4eea3b53ad4,
	0xa89aa829324d81a8, 0x966296c4f4315296,
	0xf9c3f99b3aef62f9, 0xc533c566f697a3c5,
	0x25942535b14a1025, 0x597959f220b2ab59,
	0x842a8454ae15d084, 0x72d572b7a7e4c572,
	0x39e439d5dd72ec39, 0x4c2d4c5a6198164c,
	0x5e655eca3bbc945e, 0x78fd78e785f09f78,
	0x38e038ddd870e538, 0x8c0a8c148605988c,
	0xd163d1c6b2bf17d1, 0xa5aea5410b57e4a5,
	0xe2afe2434dd9a1e2, 0x6199612ff8c24e61,
	0xb3f6b3f1457b42b3, 0x21842115a5423421,
	0x9c4a9c94d625089c, 0x1e781ef0663cee1e,
	0x4311432252866143, 0xc73bc776fc93b1c7,
	0xfcd7fcb32be54ffc, 0x0410042014082404,
	0x515951b208a2e351, 0x995e99bcc72f2599,
	0x6da96d4fc4da226d, 0x0d340d68391a650d,
	0xfacffa8335e979fa, 0xdf5bdfb684a369df,
	0x7ee57ed79bfca97e, 0x2490243db4481924,
	0x3bec3bc5d776fe3b, 0xab96ab313d4b9aab,
	0xce1fce3ed181f0ce, 0x1144118855229911,
	0x8f068f0c8903838f, 0x4e254e4a6b9c044e,
	0xb7e6b7d1517366b7, 0xeb8beb0b60cbe0eb,
	0x3cf03cfdcc78c13c, 0x813e817cbf1ffd81,
	0x946a94d4fe354094, 0xf7fbf7eb0cf31cf7,
	0xb9deb9a1676f18b9, 0x134c13985f268b13,
	0x2cb02c7d9c58512c, 0xd36bd3d6b8bb05d3,
	0xe7bbe76b5cd38ce7, 0x6ea56e57cbdc396e,
	0xc437c46ef395aac4, 0x030c03180f061b03,
	0x5645568a13acdc56, 0x440d441a49885e44,
	0x7fe17fdf9efea07f, 0xa99ea921374f88a9,
	0x2aa82a4d8254672a, 0xbbd6bbb16d6b0abb,
	0xc123c146e29f87c1, 0x535153a202a6f153,
	0xdc57dcae8ba572dc, 0x0b2c0b582716530b,
	0x9d4e9d9cd327019d, 0x6cad6c47c1d82b6c,
	0x31c43195f562a431, 0x74cd7487b9e8f374,
	0xf6fff6e309f115f6, 0x4605460a438c4c46,
	0xac8aac092645a5ac, 0x891e893c970fb589,
	0x145014a04428b414, 0xe1a3e15b42dfbae1,
	0x165816b04e2ca616, 0x3ae83acdd274f73a,
	0x69b9696fd0d20669, 0x092409482d124109,
	0x70dd70a7ade0d770, 0xb6e2b6d954716fb6,
	0xd067d0ceb7bd1ed0, 0xed93ed3b7ec7d6ed,
	0xcc17cc2edb85e2cc, 0x4215422a57846842,
	0x985a98b4c22d2c98, 0xa4aaa4490e55eda4,
	0x28a0285d88507528, 0x5c6d5cda31b8865c,
	0xf8c7f8933fed6bf8, 0x86228644a411c286,
}

```
